### PR TITLE
Respect appIdentifier from info.plist file when iOSDeploymentValidator is executed

### PR DESCRIPTION
**Pull request by Fatme**
Fixes http://teampulse.telerik.com/view#item/286588

### CODE REVIEW

File: lib/project/nativescript-project.ts [hunk 1]
```diff
@@ -55,8 +55,8 @@ export class NativeScriptProject extends frameworkProjectBaseLib.FrameworkProjec
 	public get configFiles():  Project.IConfigurationFile[] {
 		var allConfigFiles = this.$projectFilesManager.availableConfigFiles;
 		return [
-			allConfigFiles["ios-info"],
-			allConfigFiles["android-manifest"]
+			allConfigFiles[this.$projectConstants.IOS_CONFIGURARION_TEMPLATE_NAME],
```
Comment (rosen-vladimirov) [inline lib/project/nativescript-project.ts]: IOS_CONFIGURATION_TEMPLATE_NAME (configuraRion is not correct IMO)

File: lib/project/project-constants.ts [hunk 2]
```diff
@@ -9,6 +9,10 @@ export class ProjectConstants implements Project.IProjectConstants {
 	public RELEASE_PROJECT_FILE_NAME = ".release.abproject";
 	public CORE_PLUGINS_PROPERTY_NAME = "CorePlugins";
 	public CORDOVA_PLUGIN_VARIABLES_PROPERTY_NAME = "CordovaPluginVariables";
+
+	public IOS_CONFIGURARION_TEMPLATE_NAME = "ios-info";
```
Comment (rosen-vladimirov) [inline lib/project/project-constants.ts]: IOS_CONFIGURATION_TEMPLATE_NAME

File: lib/project.ts [hunk 3]
```diff
@@ -79,6 +79,18 @@ export class Project implements Project.IProject {
 		}).future<string[]>()();
 	}
 
+	public getConfigFileContent(template: string): IFuture<any> {
+		return (() => {
+			var configFile = _.find(this.projectConfigFiles, configFile => configFile.template === template);
+			if(configFile && this.$fs.exists(configFile.filepath).wait()) {
```
Comment (teobugslayer) [inline lib/project.ts]: it is a bad code practice to use exists and we should remove it from our code base. It is deprecated in both node 0.12 and io.js. See http://nodejs.org/api/fs.html#fs_fs_exists_path_callback .
Better code would be:

```
try {
  var configFileContent = this.$fs.readText(configFile.filepath).wait();
  return configFileContent;
} catch {
  return null;
}
```

**Comment by teobugslayer** [comment]
:+1:

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by ndinev** [comment]
:+1:

**Comment by Fatme** [comment]
run ci

**Pull request merged**